src/eei.h

```diff
@@ -116,8 +116,27 @@ struct EthereumInterface : ShellExternalInterface {
 };
 
 struct GasSchedule {
+  static constexpr unsigned storageLoad = 200;
   static constexpr unsigned storageStoreCreate = 20000;
   static constexpr unsigned storageStoreChange = 5000;
+  static constexpr unsigned log = 375;
+  static constexpr unsigned logData = 8;
+  static constexpr unsigned logTopic = 375;
+  static constexpr unsigned create = 32000;
+  static constexpr unsigned call = 700;
+  static constexpr unsigned copy = 3;
+  static constexpr unsigned blockhash = 800;
+  static constexpr unsigned balance = 400;
+  static constexpr unsigned stepGas1 = 2;
```

Comment:
Can you please use the names from the Yellowpapyer? `veryLow`, `low`, etc.

Reply:
Some of the gas charges in the are unused in EEI. Should they be retained in the gas schedule for the sake of yellowpaper compliance and potential later use, or trimmed off?

Reply:
Just trimmed off.